### DISCUSSION TITLE
[data-importer] Extract Field from DataObject

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#17798] [Q&A / Support] [answered]
I am using the ["Load DataObject" operator](https://github.com/pimcore/data-importer/blob/90379012c03692455260ea9b364860c1afa4824f/doc/03_Configuration/06_Mapping_Configuration/02_Transformation_Pipeline.md#list-of-available-operators) as part of my Transformation Pipeline. Once the object is loaded, how can I extract a field from that object to pass to the [data target](https://github.com/pimcore/data-importer/blob/90379012c03692455260ea9b364860c1afa4824f/doc/03_Configuration/06_Mapping_Configuration/03_Data_Target/README.md)?  

For example here I am loading a UDC object and then want to access the "ID" field on it to set on my target object:
![image](https://github.com/user-attachments/assets/b688ecc6-9ac0-4734-a073-061348cbb661)

**Comment by fashxp** [comment, marked as answer]
not possible right now. would need a custom operator.

**Reply by cancan101** [reply]
I PRed this if there is interest: https://github.com/pimcore/data-importer/pull/436
